src/System.Runtime.Extensions/tests/System/IO/Path.Combine.cs

```diff
@@ -5,279 +5,276 @@
+    {
+        //paths has 1 element
+        Verify(new String[] { "abc" });
+    }
 
-        [Fact]
-        public static void VerifyPath2Elements()
-        {
-            //paths has 2 elements
-            Verify(new String[] { "abc", "def" });
-        }
+    [Fact]
+    public static void VerifyPath2Elements()
+    {
+        //paths has 2 elements
+        Verify(new String[] { "abc", "def" });
+    }
 
-        [Fact]
-        public static void VerifyPathManyElements()
-        {
-            //paths has many elements
-            Verify(new String[] { "abc" + s_separator + "def", "def", "ghi", "jkl", "mno" });
-        }
-        [Fact]
-        public static void PathIsNullWihtoutRootedAfterArgumentNull()
-        {
-            //any path is null without rooted after (ANE)
-            CommonCasesException<ArgumentNullException>(null);
-        }
-        [Fact]
-        public static void ContainsInvalidCharWithoutRootedAfterArgumentNull()
-        {
-            //any path contains invalid character without rooted after (AE)
-            CommonCasesException<ArgumentException>("ab\"cd");
-            CommonCasesException<ArgumentException>("ab\"cd");
-            CommonCasesException<ArgumentException>("ab<cd");
-            CommonCasesException<ArgumentException>("ab>cd");
-            CommonCasesException<ArgumentException>("ab|cd");
-            CommonCasesException<ArgumentException>("ab\bcd");
-            CommonCasesException<ArgumentException>("ab\0cd");
-            CommonCasesException<ArgumentException>("ab\tcd");
-        }
-        [Fact]
-        public static void ContainsInvalidCharWithRootedAfterArgumentNull()
-        {
-            //any path contains invalid character with rooted after (AE)
-            CommonCasesException<ArgumentException>("ab\"cd", s_separator + "abc");
-            CommonCasesException<ArgumentException>("ab<cd", s_separator + "abc");
-            CommonCasesException<ArgumentException>("ab>cd", s_separator + "abc");
-            CommonCasesException<ArgumentException>("ab|cd", s_separator + "abc");
-            CommonCasesException<ArgumentException>("ab\bcd", s_separator + "abc");
-            CommonCasesException<ArgumentException>("ab\0cd", s_separator + "abc");
-            CommonCasesException<ArgumentException>("ab\tcd", s_separator + "abc");
-        }
+    [Fact]
+    public static void VerifyPathManyElements()
+    {
+        //paths has many elements
+        Verify(new String[] { "abc" + s_separator + "def", "def", "ghi", "jkl", "mno" });
+    }
+    [Fact]
+    public static void PathIsNullWihtoutRootedAfterArgumentNull()
+    {
+        //any path is null without rooted after (ANE)
+        CommonCasesException<ArgumentNullException>(null);
+    }
+    [Fact]
+    public static void ContainsInvalidCharWithoutRootedAfterArgumentNull()
+    {
+        //any path contains invalid character without rooted after (AE)
+        CommonCasesException<ArgumentException>("ab\"cd");
```

Comment:
Not actionable right now, but we're going to need some way to denote that certain tests are platform-independent while other tests are platform-specific for a particular platform, and potentially a way to pass through information about the platform being tested.  For the former, we could probably use an attribution scheme similar to what we have for OuterLoop, e.g. no attribution means platform-independent, and then an [OS("Windows")], [OS("Unix")], etc. attribution to denote that a test should only be run on specific platforms.

(I mention this here because the list of invalid characters differs by platform; most of these are perfectly valid on Unix.)